website/content/docs/install/_index.md

```diff
@@ -42,6 +42,9 @@ Next you can install the binary on your system using the following `tar` command
 tar xzf bctl_<os>_<arch>.tar.gz -C /usr/local/bin/
 ```
 
+### Upgrading the bctl binary version
```

Comment:
You should be able to just rerun the script and it will override the existing install. Did you run into an issue?

Reply:
Thanks for clarifying how the installer script works. I haven't tested or tried that out. I wasn't sure that was the case with the installer script. I was hoping we could make it clear to the end user in our documentation that inplace upgrading the bctl binary was possible (without needing to uninstall boundless first).

Should we perhaps mention the in-place upgrading aspect in our installer portion of this documentation? "You may install or upgrade to the latest Boundless with the following script."

Reply:
I'm actually debating if it makes sense to start breaking this page into separate sections. The page title is `Install` but it now contains much more than just the install methods.